addons/cookoff/XEH_postInit.sqf

```diff
@@ -46,14 +46,21 @@ GVAR(cacheTankDuplicates) = call CBA_fnc_createNamespace;
 
 // secondary explosions
 ["AllVehicles", "killed", {
+    if ((_this select 0) isKindOf "StaticWeapon") exitWith {};
```

Comment:
just add `"StaticWeapon"` to the array where `"Man"` already is excluded. (L52/57)